datasets/cvpp/yaml.yaml

```diff
@@ -0,0 +1,23 @@
+name: cvpp.lt
+
+#Paieskos puslaio formatas
+#http://cvpp.lt/index.php?limitstart=10
+#http://cvpp.lt/index.php?limitstart=20
+#...
+
+#Vieno iraso puslapio adreso pavyzdys
+#http://cvpp.lt/index.php?option=com_vpt&theme=new&task=view&tender_id=220521
+#svarbus tender_id
+
+#Atrodo, paprasciausiai ieskoti per paieskos puslapi atsizvelgiant i data, is paieskos puslapio gauti tender_id ir tada siusti.
+
+crowler:
+    start-urls: http://cvpp.lt/
+    navigate-urls: ?
+    download-urls: ?
```

Comment:
Pasižiūrėjęs kaip veikia https://www.kimonolabs.com/ galvoju, kad gal `navigate-urls` ir `download-urls` vis dėl to nereikia. Geriau tai patikėti `scraper` daliai. T.y. vietoje to, kad `crawler` dalyje nurodyti visas nuorodas, kuriomis reikia sekti, geriau `scraper` dalyje, tiksliai nurodyti kokiuose HTML elementuose tos nuorodos yra. Taip gausime daugiau lankstumo ir efektyvumo. Ką manai?